BrewMesModulesParent/machine/src/main/java/com/brewmes/machine/MachineServiceImpl.java

```diff
@@ -0,0 +1,259 @@
+    private Map<String, OpcUaClient> opcUaClients = new HashMap<>();
+
+
+    //If the program is restarted manually or due to a crash this method will assure that machines which were autobrewing before start doing it again
+    @PostConstruct
+    private void initializeAutoBrew() {
+        List<Connection> list = getConnections();
+
+        for (Connection connection : list) {
+            if (connection.isAutobrewing()) {
+                startAutoBrew(connection.getId());
+            }
+        }
+    }
+
+    private void connectToMachine(String connectionID) {
+        if (!opcUaClients.containsKey(connectionID)) {
+            //Get connection object from database based on id
+            Connection connection = getConnection(connectionID);
+
+            //Create and add connection to the map
+            try {
+                opcUaClients.put(connectionID, getOpcUaClient(connection.getIp()));
+            } catch (InterruptedException e) {
+                logger.log(Level.SEVERE, e.getMessage(), e);
+                Thread.currentThread().interrupt();
+            } catch (ExecutionException | UaException e) {
+                logger.log(Level.SEVERE, e.getMessage(), e);
+            }
+        }
+    }
+
+    private OpcUaClient getOpcUaClient(String ipAddress) throws InterruptedException, ExecutionException, UaException {
+        //get all endpoints from the machine
+        List<EndpointDescription> endpoints = DiscoveryClient.getEndpoints(ipAddress).get();
+
+        String[] ipAddressArray = ipAddress.split(":");
+        EndpointDescription configPoint = EndpointUtil.updateUrl(endpoints.get(0), ipAddressArray[1].substring(2), Integer.parseInt(ipAddressArray[2]));
+
+        //loading endpoints into configuration
+        OpcUaClientConfigBuilder cfg = new OpcUaClientConfigBuilder();
+        cfg.setEndpoint(configPoint);
+
+        //setting up machine client with config
+        OpcUaClient opcUaClient = OpcUaClient.create(cfg.build());
+
+        //connecting machine
+        opcUaClient.connect().get();
+        return opcUaClient;
+    }
+
+    private void sendChanges(String connectionID) {
+        try {
+            NodeId cmdChangeRequest = CommandNodes.EXECUTE_MACHINE_COMMAND.nodeId;
+            opcUaClients.get(connectionID).writeValue(cmdChangeRequest, DataValue.valueOnly(new Variant(true))).get();
+        } catch (InterruptedException | ExecutionException e) {
+            logger.log(Level.SEVERE, e.getMessage(), e);
+            Thread.currentThread().interrupt();
+        }
+    }
+
+    /**
+     * Takes a command and sends it to the desired machine
+     *
+     * @param command      The {@code command} that should be sent to the machine
+     * @param connectionID The ID of the machine that should receive the {@code command}
+     * @return Returns {@code true} if the control {@code command} was successfully sent, returns {@code false} if the control {@code command} failed
+     */
```

Comment:
```suggestion
    /**
     * {@inheritdoc}
     */
```
Either inherit the javadoc from the interface or just remove it entirely, no reason to copy it.